commands/raid_from_webhook.php

```diff
@@ -125,12 +124,12 @@ function isPointInsidePolygon($point, $polygon) {
                     gym_id LIKE :gym_id
             ';
             $statement = $dbh->prepare( $query );
-            $statement->bindValue(':lat', $gym_lat, PDO::PARAM_STR);
-            $statement->bindValue(':lon', $gym_lon, PDO::PARAM_STR);
-            $statement->bindValue(':gym_name', $gym_name, PDO::PARAM_STR);
-            $statement->bindValue(':ex_gym', $gym_is_ex, PDO::PARAM_INT);
-            $statement->bindValue(':img_url', $gym_img_url, PDO::PARAM_STR);
-            $statement->bindValue(':gym_id', $gym_id, PDO::PARAM_STR);
+            $statement->bindValue(':lat', $gym_lat);
+            $statement->bindValue(':lon', $gym_lon);
+            $statement->bindValue(':gym_name', $gym_name);
+            $statement->bindValue(':ex_gym', $gym_is_ex);
+            $statement->bindValue(':img_url', $gym_img_url);
+            $statement->bindValue(':gym_id', $gym_id);
```

Comment:
bindValue vs. bindParam:
https://stackoverflow.com/a/5077108

Since it shouldn’t make a difference, we should always either use bindValue or bindParam in the whole code.
Don’t know if we can take  advantage of either bindParam or bindValue, but as we almost always do prepare, bind, execute we should avoid confusion and just use either bindParam or bindValue as long as we don’t take advantage of one of them.

Reply:
Yeah, that was a snafu on my part. But, I will convert this to an execute() style instead which eliminates the whole problem.